docs/ingestion/tasks.md

```diff
@@ -98,6 +101,14 @@ For some task types, the indexing task can wait for the newly ingested segments
 |`segmentAvailabilityConfirmed`|Whether all segments generated by this ingestion task had been confirmed as available for queries in the cluster before the task completed.|
 |`segmentAvailabilityWaitTimeMs`|Milliseconds waited by the ingestion task for the newly ingested segments to be available for query after completing ingestion was completed.|
 
+#### Ingested Intervals
```

Comment:
Docs look good. Only nit is to use sentence case for subheadings: `Ingested intervals`